pkg/component/machinecontrollermanager/templates/crd-machinedeployments.tpl.yaml

```diff
@@ -4,8 +4,7 @@ kind: CustomResourceDefinition
 metadata:
   annotations:
     controller-gen.kubebuilder.io/version: v0.9.2
-  labels:
-    gardener.cloud/deletion-protected: "true"
```

Comment:
Revert this and similar removals of the `gardener.cloud/deletion-protected` label.